x/clp/keeper/querier.go

```diff
@@ -23,35 +22,29 @@ func NewQuerier(keeper Keeper, legacyQuerierCdc *codec.LegacyAmino) sdk.Querier
 		case types.QueryAssetList:
 			return queryAssetList(ctx, path[1:], req, keeper, legacyQuerierCdc)
 		case types.QueryLPList:
-			return queryLPList(ctx, path[1:], req, keeper, legacyQuerierCdc)
+			return queryLPList(ctx, path[1:], req, keeper, legacyQuerierCdc, querier)
 		case types.QueryAllLP:
-			return queryAllLP(ctx, path[1:], keeper, legacyQuerierCdc)
+			return queryAllLP(ctx, path[1:], keeper, legacyQuerierCdc, querier)
```

Comment:
Get pools should also be hooked up to go through query server.

Reply:
they are